tests/python/serve/test_serve_engine.py

```diff
@@ -14,6 +14,9 @@
 )
 from mlc_llm.serve.engine import ModelInfo
 
+model_path = "dist/Llama-2-7b-chat-hf-q0f16-MLC"
+model_lib_path = "dist/Llama-2-7b-chat-hf-q0f16-MLC/Llama-2-7b-chat-hf-q0f16-MLC-cuda.so"
+
```

Comment:
Can we revert the changes in this file? Given this test is not rather standard (like used in CI), I personally prefer a finer control of the paths in each function.